src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -288,16 +311,16 @@ private static bool IsAccessorListWithoutAccessorsWithBlockBody(SyntaxNode? node
                 accessorList.Accessors.All(a => a.Body == null);
 
         private static bool IsAccessorListFollowedByInitializer([NotNullWhen(true)] SyntaxNode? node)
-            => node is AccessorListSyntax accessorList &&
-                node.Parent is PropertyDeclarationSyntax property &&
-                property.Initializer != null;
+            => node is AccessorListSyntax { Parent: PropertyDeclarationSyntax { Initializer: not null } };
```

Comment:
In general, please try to keep style-only changes like these to a minimum in the compiler layer. No need to undo any such changes already in the PR.

Reply:
Originally I only removed `accessorList` as it was never used, but Cyrus then suggested to use pattern matching and I accepted it. Ok, will try to avoid such changes later.

Also I have an idea to convert all multiline strings is `SyntaxNormalizerTests` to raw form in order to make them actually readable in a separate PR after this one is merged. Would such change be considered as style-only and rejected?

Reply:
> Also I have an idea to convert all multiline strings is `SyntaxNormalizerTests` to raw form in order to make them actually readable in a separate PR after this one is merged

I personally would be fine with this. @jcouv as long as you're also fine with this, I'd say @DoctorKrolic can go ahead and open a subsequent PR to convert the `@""` strings in that file to `""" """`.

Reply:
i would be fine with this.  they're tests, so there's no risk.  and test-strings were a primary use case for raw-strings :)